### PR TITLE
connect to a redis cache in prod/test

### PR DESCRIPTION
**Pull request by NoamNissan**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -33,7 +33,14 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 if (env === 'production' || env === 'test') {
     var RedisStore = require('connect-redis')(session);
-    var redisClient = redis.createClient();
+    var redisClient = redis.createClient(
```
Comment (iTZAvishay) [inline app.js:36]: should anyone even use it?

Comment (NoamNissan) [reply app.js:36]: not in local setup. I have added a redis cache only in prod

File: config/config.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 var fs = require('fs');
-
+dotenv = require('dotenv');
```
Comment (iTZAvishay) [inline config/config.js:2]: this should only happen once in the project, if you don't want a specific file you can change 

`npm start` to use `-r` (https://nodejs.org/api/cli.html#cli_r_require_module)

likeso:
`node -r dotenv/config ./bin/www`

if you do that, don't forget to change `npm dev` as well

Comment (NoamNissan) [reply config/config.js]: fixed

File: app.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+dotenv = require('dotenv');
```
Comment (iTZAvishay) [inline app.js:1]: I'd love a simple one-liner `require('dotenv').config()`

Comment (NoamNissan) [reply app.js]: you got it

**Pull request merged**